.github/workflows/gerrit-compose-required-tox-verify.yaml

```diff
@@ -92,7 +92,7 @@ jobs:
         tox-env: ${{ fromJSON(inputs.TOX_ENVS) }}
 
     steps:
-      - name: Gerrit Checkout
+      - name: Gerrit checkout
```

Comment:
Is the change in casing on purpose or accidental?

Reply:
On purpose. When I added the other step, I noticed that the casing was inconsistent between the steps already there, so I wanted to make it consistent across all steps. All steps are now capitalized as sentences rather than titles.